public/TrafficStats.php

```diff
@@ -137,10 +138,11 @@ public function doPOST ($startpt, $endpt, $time,$request_data=null)
 
         $retVal = new stdClass;
         $retVal->query = $aboutQuery;
-        $retVal->info = $infoObj;
         $retVal->results = $qRes;
+        $retVal->info = $infoObj;
 
-        return $retVal;
+        return new ApiResult($retVal);
```

Comment:
Doesn't this create an object that looks like
```json
{
  "results": {
      "query": "q",
      "info": "i",
      "results": "r"
    }
}
```

We want to create the API result with the $qRes, and $aboutQuery in the debug spot, and then add the $infoObj on as a third value, THEN return the result.